### ISSUE TITLE
How to get val loss and save best checkpoint in the ssd_mobilenet_v1 tutorial 

### ISSUE THREAD

**Issue by JaviMota** [tensorflow/models#10801]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [ x ] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the documentation with the issue

https://colab.research.google.com/github/google-coral/tutorials/blob/master/retrain_detection_qat_tf1.ipynb#scrollTo=ImucOu0qgMv_

## 2. Describe the issue

Hi, Im doing transfer learning with the ssd mobilenet v1 and I would like to know how can I get the val loss during the training and how to save only the best checkpoint according to the best val loss. I only get the loss and by default the checkpoints saved are the first one and the last one.

Thank you.

**Comment by aymanmostafa11** [comment]
Hello, I believe the repo you're looking for is [google-coral/tutorials](https://github.com/google-coral/tutorials) which contains the referred to tutorial.
